src/managers/GuildManager.js

```diff
@@ -128,24 +128,29 @@ class GuildManager extends BaseManager {
     return super.resolveID(guild);
   }
 
+  /**
+   * Options used to create a guild.
+   * @typedef {Object} GuildCreateOptions
+   * @param {number} [afkChannelID] The ID of the AFK channel
+   * @param {number} [afkTimeout] The AFK timeout in seconds
+   * @param {PartialChannelData[]} [channels] The channels for this guild
+   * @param {DefaultMessageNotifications} [defaultMessageNotifications] The default message notifications
+   * for the guild
+   * @param {ExplicitContentFilterLevel} [explicitContentFilter] The explicit content filter level for the guild
+   * @param {BufferResolvable|Base64Resolvable} [icon=null] The icon for the guild
+   * @param {string} [region] The region for the server, defaults to the closest one available
+   * @param {PartialRoleData[]} [roles] The roles for this guild,
+   * the first element of this array is used to change properties of the guild's everyone role.
+   * @param {number} [systemChannelID] The ID of the system channel
+   * @param {SystemChannelFlagsResolvable} [systemChannelFlags] The flags of the system channel
+   * @param {VerificationLevel} [verificationLevel] The verification level for the guild
```

Comment:
typedefs have `property`'s, not `param`'s

Reply:
This is what happens when you work at the time when you should be sleeping, will fix soon 👍

Reply:
xD all good